Dockerfile.xpu

```diff
@@ -3,20 +3,26 @@ FROM intel/oneapi-basekit:2024.1.0-devel-ubuntu20.04
 RUN wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB | gpg --dearmor | tee /usr/share/keyrings/intel-oneapi-archive-keyring.gpg > /dev/null && \
     echo "deb [signed-by=/usr/share/keyrings/intel-oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main " | tee /etc/apt/sources.list.d/oneAPI.list && \
     chmod 644 /usr/share/keyrings/intel-oneapi-archive-keyring.gpg && \
-    rm /etc/apt/sources.list.d/intel-graphics.list && \
```

Comment:
This seems unrelated, but `intel-graphics.list` is not present and the build fails otherwise, trying to remove a non-existing file.

Reply:
You could add `-f` which will result in 0 exit code regardless of whether the file exists